api/dp-openapi.yaml

```diff
@@ -330,6 +353,72 @@ paths:
         default:
           $ref: '#/components/responses/UnexpectedError'
 
+  /projects/{projectid}/holdstates:
+    get:
+      tags:
+      - project
+      description: Returns a list of hold states for the project
+      parameters:
+      - name: projectid
+        in: path
+        description: ID of project
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: List of current project hold states
+          content:
+            application/json:
+              schema:
+                items:
+                  type: string
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '429':
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+    put:
+      tags:
+      - project
+      description: Sets the holdable states for a project
+      parameters:
+      - name: projectid
+        in: path
+        description: ID of project
+        required: true
+        schema:
+          type: string
+      requestBody:
+        description: List of holdable states to set
+        required: true
+        content:
+          application/json:
+            schema:
```

Comment:
In some places you put type: array on schema and others not. Is it intentional?

Reply:
That was not intentional and I'm kinda surprised the Swagger editor didn't flag it. I've added the `type: array` to these two places, thanks for catching it.